security/firely-auth/firely-auth-settings.rst

```diff
@@ -459,6 +459,8 @@ The content of a disclaimer is user-defined and can be expressed in a `liquid te
 For each disclaimer a checkbox is rendered in the UI by Firely Auth on the disclaimer page. A description shown next the checkbox can be defined for each disclaimer.
 Firely Auth will automatically fill out placeholders defined in the liquid template based on static properties defined as ``TemplateProperties``.
 
+For versioning, the ``Id`` property can be used, like using ``GeneralTermsV1`` and then changing it to ``GeneralTermsV2`` if needed.
```

Comment:
Could you add an example here and also add a note on what happens to the existing consent grants?

Reply:
updated